modules/nf-core/hifiasm/main.nf

```diff
@@ -6,9 +6,7 @@ process HIFIASM {
         exit 1, "This version of HIFIASM module does not support Conda. Please use Docker / Singularity / Podman instead."
```

Comment:
no conda here

Reply:
Yes, thanks for spotting. Something went wrong with adding files in the previous commit.